frameworks/cassandra/tests/conftest.py

```diff
@@ -1,10 +1,15 @@
 import pytest
+import sdk_login
 import sdk_repository
 import sdk_security
 
 
 @pytest.fixture(scope='session')
-def configure_universe():
+def configure_login():
```

Comment:
I don't want to do top-level because I like thinking of the frameworks as separate entities as much as possible. If anything, I'd rather go back to how I had the setup working originally:

```
@pytest.fixture(scope='session')
def configuire_session():
  try:
    login_config = sdk_login.configure_login()
    repo_config = sdk_repository.configure_repository()
    security_config = sdk_security.configure_security('cassandra')
    yield
  finally:
    sdk_security.cleanup_security(security_config)
    sdk_repository.cleanup_repository(repo_config)
    sdk_login.cleanup_login(login_config)
```

I think that makes the setup clean, explicit, and minimizes the boilerplate, but I'm not tied to any implementation. I'll leave it as-is as a matter of inertia.